woodchipper/decorator.py

```diff
@@ -27,16 +31,18 @@ def _get_key_or_attr(obj, name: str, missing=missing):
             return missing
 
 
-def _pluck_value(obj, str_path: str, delimiter=".") -> Union[str, _Missing]:
+def pluck_value(obj, str_path: str, delimiter=".") -> Union[str, Missing]:
+    """Dig into an object or dict Using a JMSE-like path syntax. Return Missing if value cannot be found. Only
+    supports mapping-like behavior so cannot search sequences."""
     if not str_path:
-        return str(obj)
+        return obj
```

Comment:
In order to meet your typing contract, you have to return a string. If we had a static type checker running, it would berate you for this and make all sorts of unseemly insinuations as to the quality of your character and intelligence. But it's a cruel, cruel piece of software.